client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -127,8 +132,10 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                       lb-posts-list
                       lb-posts-blog-id="blog._id"
                       lb-posts-status="open"
-                      lb-posts-allow-unpublish="true"
-                      lb-posts-allow-reordering="true"
+                      lb-posts-allow-unpublishing="blogSecurityService.canPublishAPost"
```

Comment:
all this mapping looks ugly,
pass the `blogSecurityService` to the directive or
a convention to make all the methods name starting with `can` to merge in the directive